src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -66,11 +66,21 @@ private UserStore(PersistentStorageAgent persistentStorageAgent) {
     users = new ArrayList<>();
 
     // hard-coded initial Admin:
-    // TODO(JW): Merge "the creation process" from RegisterServlet to here.
-    String hashedPassword = BCrypt.hashpw("AdminPass01", BCrypt.gensalt());
-    User initialAdmin = new User(UUID.randomUUID(), "Admin01", hashedPassword, Instant.now());
-    initialAdmin.setAdmin(true);
-    users.add(initialAdmin);
+    this.addUser("Admin01", "AdminPass01", true);
+  }
+
+  /**
+   * Add a new user to the current set of users known to the application. This should only be called
+   * to add a new user, not to update an existing user.
+   */
+  public void addUser(String username, String password, boolean adminStatus) {
+    String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
+    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
+    if (adminStatus) {
```

Comment:
This could be written more simply as `user.setAdmin(adminStatus)` since it works whether `adminStatus` is true or false.

Reply:
Indeed, thank you!